### PR TITLE
Add generate sales report solution

### PR DESCRIPTION
**Pull request by aliciawilliams**

### CODE REVIEW

File: generate-sales-report/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+`Constants.gs` replacing `YOUR_SLIDES_ID` and maintaining 
+the quotes.
+1. Save the changes by navigating to **File > Save**.
+
+
+### Generate a new report
+
+1. Navigate to the Generator tab of your spreadsheet.
+1. Choose an Account Name and Region in drop-down cells.
+1. Navigate to the Data Results tab and click **Refresh**
```
Comment (asrivas) [inline generate-sales-report/README.md]: Can we add a screenshot here? We can probably crop & reuse the images from the blog post

File: generate-sales-report/src/Code.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+  stageSlide(slides[2], presentation, sheet, tempSheet, numRows);
+  businessSlide(slides[3], presentation, sheet, tempSheet, numRows);
+  topSlide(slides[4], sheet);
+  nearSlide(slides[5], sheet);
+  needsSlide(slides[6], sheet);
+
+  spreadsheet.deleteSheet(tempSheet);
+}
+
+function introSlide(slide, presentation) {
```
Comment (asrivas) [inline generate-sales-report/src/Code.js]: can you either add function comments OR make them private functions by appending '_' to the name:
function introSlide_(.....

https://developers.google.com/apps-script/guides/html/communication#private_functions

File: generate-sales-report/src/Code.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Default values
+var ACCOUNT_NAME = 'Acme';
```
[the remark this answers is not in the document]
Comment (aliciawilliams) [reply generate-sales-report/src/Code.js]: changed to accountName, plus mentions in code

File: generate-sales-report/README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+for many enterprises. But automation can help eliminate the 
+repetition inherent in these periodic obligations. 
+If you find yourself needing to analyze and present findings 
+from large sets of stored data, consider using automation 
+to help streamline your reporting. 
+
+This solution creates a tool that connects to sales data in 
+BigQuery, an analytics data warehouse, from directly within 
+Google Sheets. A configuration sheet allows a user to provide 
+parameters for the report, such as the Account Name and
```
[the remark this answers is not in the document]
Comment (aliciawilliams) [reply generate-sales-report/README.md]: Changed to quotes here, and also in the Try It section

File: generate-sales-report/README.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+### Set up the configuration spreadsheet
+
+1. Make of copy of the template spreadsheet [here][spreadsheet].
+1. From the spreadsheet, open the script editor by selecting 
+**Tools > Script editor**.
+1. Copy and paste your Slides document ID into line 1 of 
+`Constants.gs` replacing `YOUR_SLIDES_ID` and maintaining 
+the quotes.
+1. Save the changes by navigating to **File > Save**.
+
```
[the remark this answers is not in the document]
Comment (aliciawilliams) [reply generate-sales-report/README.md]: Removed all the extra lines in the README

File: generate-sales-report/README.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+_Bring sales data from your data warehouse directly into
+ Google Sheets, and then automatically generate presentations 
+ to summarize wins and opportunities for particular regions 
+ and customers._
+
+Last updated: November, 2019
+
+Annual and quarterly reports are a standard part of business 
+for many enterprises. But automation can help eliminate the
```
[the remark this answers is not in the document]
Comment (aliciawilliams) [reply generate-sales-report/README.md]: Changed "But" to "Luckily," which I think sounds better.

File: generate-sales-report/README.md (hunk 1 above)
Comment (aliciawilliams) [reply generate-sales-report/README.md]: These steps are what is shown in the gif that I have in the intro section...do you want a screenshot of one of the particular steps?

Comment (aliciawilliams) [reply generate-sales-report/README.md]: Gif is there!

File: generate-sales-report/src/Code.js (hunk 2 above)
Comment (aliciawilliams) [reply generate-sales-report/src/Code.js]: all functions except generateReport() are now private

**Comment by aliciawilliams** [comment]
@asrivas I have used the linter and updated the code in Github and in the template document. Can you please review and merge?

**Pull request merged**